### DISCUSSION TITLE
Checks on PRs stopped working

### DISCUSSION THREAD

**Discussion by annayeomans** [community/community#26701] [Actions]
<p>Yesterday our linting checks stopped working on PRs as well as another check we had integrated with another tool we are using.<br>
We changed nothing. Any ideas on how to debug and what could be the issue?</p>

**Comment by alikr123** [comment]
<p>Github actions has been unstable lately.<br>
Closing and opening the PRs worked for us.</p>

**Comment by ashishmgofficial** [comment]
<p>Same here.</p>
<p><code>${{ github.event.pull_request.merged == 'true' }}</code>  always returns <code>false</code> now for me</p>

**Comment by ghost** [comment]
<p>contains( ${{ github.event.pull_request.labels.*.name }}, ‘security’) always returns “true” no matter i will label or not</p>

**Comment by jricciardi** [comment]
<p>Hi there <a class="mention" href="/annayeomans">@annayeomans</a>! I wanted to follow up. Did this problem ever resolve, or were you able to troubleshoot it?</p>

**Comment by airtower-luna** [comment]
<aside class="quote no-group" data-username="Tirumalesh-Yeligar_ghub" data-post="4" data-topic="240906" data-full="true">
<div class="title">
<div class="quote-controls"></div>
 Tirumalesh-Yeligar_ghub:</div>
<blockquote>
<p>contains( ${{ github.event.pull_request.labels.*.name }}, ‘security’) always returns “true” no matter i will label or not</p>
</blockquote>
</aside>
<p>That’s because only <em>part</em> of your expression is enclosed in <code>${{ .. }}</code>. Only the part between the curly braces is evaluated as an expression, the rest is just a string. And a non-empty string is always <code>true</code>.</p>
<p>Try:</p>
<pre><code class="lang-auto">${{ contains(github.event.pull_request.labels.*.name, 'security') }}
</code></pre>

**Comment by annayeomans** [comment, marked as answer]
<p>Hi! Yes, It just resolved on its own.</p>

**Comment by jricciardi** [comment]
<p><a class="mention" href="/annayeomans">@annayeomans</a> Love to hear it! :raised_hands:</p>
